analytical/tests/test_tag_facebook_pixel.py

```diff
@@ -0,0 +1,87 @@
+"""
+Tests for the Facebook Pixel template tags.
+"""
+from django.http import HttpRequest
+from django.template import Context
+from django.test import override_settings
+
+from analytical.templatetags.facebook_pixel import FacebookPixelHeadNode, FacebookPixelBodyNode
+from analytical.tests.utils import TagTestCase
+from analytical.utils import AnalyticalException
+
+
+expected_head_html = """\
```

Comment:
Is the backslash needed?

Reply:
This is the conventional way (as far as I know) to avoid inserting an unwanted extra empty line in the beginning of a block string literal. So for example:

```Python console
>>> """\
... One
... Two
... """.splitlines()
['One', 'Two']

>>> """
... One
... Two
... """.splitlines()
['', 'One', 'Two']
```

In this case, the purpose is just to have nice clean HTML output. :)

Reply:
Indeed, this is convention, although I cannot find an authoritative link at the moment.

Reply:
[This StackOverflow question](https://stackoverflow.com/q/2504411/444705) has some examples of this pattern (also in conjunction with `dedent()` for indentation stripping).